_partials/developer-materials-results.html.slim

```diff
@@ -41,7 +41,7 @@ div
             | Product
           td.skill Skill Level
       tbody
-        tr.material(ng-repeat="item in data.displayedMaterials" class="get-started-placeholder-{{item.fields.sys_type}}")
+        tr.material(ng-repeat="item in data.displayedMaterials | middlewareOnly" class="get-started-placeholder-{{item.fields.sys_type}}")
```

Comment:
And I recommend to move the `sys_type:video` from `query` to filter. This means from `&query=sys_type:video ...` to `&sys_type=video&query= ...`. Check my URL suggestion above.